examples/fluids/navierstokes.c

```diff
@@ -256,7 +256,8 @@ static int VectorPlacePetscVec(CeedVector c, Vec p) {
   ierr = CeedVectorGetLength(c, &mceed); CHKERRQ(ierr);
   ierr = VecGetLocalSize(p, &mpetsc); CHKERRQ(ierr);
   if (mceed != mpetsc) SETERRQ2(PETSC_COMM_SELF,PETSC_ERR_ARG_INCOMP,
-                                  "Cannot place PETSc Vec of length %D in CeedVector of length %D",mpetsc,mceed);
+                                "Cannot place PETSc Vec of length %D in CeedVector of length %D",
```

Comment:
Do we know why `make-style` would change this line back to having two spaces after the `(`, but it wouldn't for similar macro calls, such as `SETERRQ4` call on line 215?

Reply:
I don't know.